lib/cartopy/mpl/geoaxes.py

```diff
@@ -784,6 +796,183 @@ def stock_img(self, name='ne_shaded'):
         else:
             raise ValueError('Unknown stock image %r.' % name)
 
+    def background_img(self, name='ne_shaded', resolution='low', extent=None,
+                       cache=False):
+        """
+        Adds a background image to the map, from a selection of pre-prepared
+        images held in a directory specified by the CARTOPY_USER_BACKGROUNDS
+        environment variable. That directory is checked with
+        func:`self.read_user_background_images` and needs to contain a JSON
+        file which defines for the image metadata.
+
+        Kwargs:
+
+            * name - the name of the image to read according to the contents
+                     of the JSON file. A typical file might have, for instance:
+                     'ne_shaded' : Natural Earth Shaded Relief
+                     'ne_grey' : Natural Earth Grey Earth
+
+            * resolution - the resolution of the image to read, according to
+                           the contents of the JSON file. A typical file might
+                           have the following for each name of the image:
+                           'low', 'med', 'high', 'vhigh', 'full'.
+
+            * extent - using a high resolution background image, zoomed into
+                       a small area, will take a very long time to render as
+                       the image is prepared globally, even though only a small
+                       area is used. Adding the extent will only render a
+                       particular geographic region. Specified as
+                       [longitude start, longitude end,
+                        latitude start, latitude end].
+
+                       e.g. [-30.0, 10.0, -30.0, 10.0] for Ascension Island
+                       or [167.0, 193.0, 47.0, 68.0] to cross the date line.
+
+            * cache - logical flag as to whether or not to cache the loaded
+                      images into memory. The images are stored before the
+                      extent is used.
+        """
+        # read in the user's background image directory:
+        if len(_USER_BG_IMGS) == 0:
+            self.read_user_background_images()
+        import os
+        bgdir = os.getenv('CARTOPY_USER_BACKGROUNDS')
+        if bgdir is None:
+            bgdir = os.path.join(config["repo_data_dir"],
+                                 'raster', 'natural_earth')
+        # now get the filename we want to use:
+        try:
+            fname = _USER_BG_IMGS[name][resolution]
+        except KeyError:
+            msg = ('Image "{}" and "resolution "{}" are not present in '
```

Comment:
Do you need the `"` before resolution? It doesn't have a corresponding closing quote.